src/host/chuck_main.cpp

```diff
@@ -492,6 +493,8 @@ bool go( int argc, const char ** argv )
             // blocking removed, ge: 1.3.5.3
             // else if( !strcmp(argv[i], "--blocking") )
             //     block = TRUE;
+            else if( !strcmp(argv[i], "--syntax-check-only") )
+                syntax_check_only = TRUE;
```

Comment:
I think this works as-is already, but it might be faster to add the following to this if-statement:
`g_enable_realtime_audio = FALSE;`
(and then would need braces for the if statement).

It should then be much faster without initializing the audio system.

I am pretty sure that our automated tests don't launch the audio callback, but there was a while when there was a bug and they did; during that time the testing was a lot slower. So if you are trying to do this in an automated way behind the scenes of VSCode, then not starting the audio callback should help a lot. 

(Similar optimizations could be done for not initializing the Hid and Midi managers, but that would involve a lot more code complexity. And, I don't think our tests bother with disabling the Hid or Midi managers, so probably not worth any effort there.)